frontend/src/store/wallet.ts

```diff
@@ -132,9 +135,6 @@ export default function useWalletStore() {
             window.location.reload();
           });
           wallet.provider.on('chainChanged', () => {
-            if (wallet.label === 'WalletConnect' && !lastWallet.value) {
-              setLastWallet('WalletConnect');
-            }
```

Comment:
This was modified recently in https://github.com/ScopeLift/umbra-protocol/pull/559#discussion_r1253300467, why was it needed then but not now?

Reply:
When logging into wallet connect the page would reload before the last wallet was set which created a state where the user seemed to be logged out after the page reloaded. I tested and I wasn't able to create the issue the required this after upgrading the packages.